src/main/features/claimant-response/routes/receipt.ts

```diff
@@ -1,21 +1,10 @@
 import * as express from 'express'
 import { Paths } from 'claimant-response/paths'
 import { ErrorHandling } from 'shared/errorHandling'
-import { DocumentsClient } from 'documents/documentsClient'
-import { Claim } from 'claims/models/claim'
-import { DownloadUtils } from 'utils/downloadUtils'
-
-const documentsClient: DocumentsClient = new DocumentsClient()
 
 /* tslint:disable:no-default-export */
 export default express.Router()
   .get(Paths.receiptReceiver.uri,
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
-      const { externalId } = req.params
-      const claim: Claim = res.locals.claim
-
-      const filename: string = `${claim.claimNumber}-county-court-judgement`
-      const pdf: Buffer = await documentsClient.getCountyCourtJudgmentPDF(externalId, res.locals.user.bearerToken)
-
-      DownloadUtils.downloadPDF(res, pdf, filename)
+      throw Error('CCJ document cannot be downloaded')
```

Comment:
is there any need to have this route at all if it is not used by anything?

Reply:
The only reason I kept the file was that it will be easier to revert the changes if the need be. I am happy to delete the file if that is a better option

Reply:
Please do remove the file. we'll not be making documents available for several months